### PR TITLE
Added an example Gson serialization to store or load a given Series t…

### PR DESCRIPTION
**Pull request by MarcDahlem**
…o json

Fixes #.

Changes proposed in this pull request:
- added an example Gson de-/serialization for BarSeries

- [x] added an entry with related ticket number(s) to the unreleased section of `CHANGES.md`

### CODE REVIEW

**Comment by MarcDahlem** [comment]
Oh well, that failed test run is interesting. It randomly picked one index, which does not need the bigInteger precession.
As I also worked with the Interface "Number" in the Gson-Wrapper-Classes, Gson treats the number as double value instead of BigDecimals.

How should I proceed here?
a) Is the (Base)BarSeries able to contain some bars as BigDecimal and others as double or even integer? (This would mean I could just fix the tests).
b) Or should I force Gson to always produce BigDecimals while adding a new bar to the created bar series?

**Comment by MarcDahlem** [comment]
Okay, nevermind.
The test alone runs fine on all elements; only if executed with all other tests together it didnt work.

Someone thought that changing the defaultfunction in the BarSeriesTest (main-method in ta4jexamples/barSeries/BuildBarSeries.java ) was a good idea...

I made this BuildBarSeries reset the defaultFunction to ensure correct behaviour for all tests running afterwards

**Review by team172011 — CHANGES_REQUESTED** [review]
Thanks for the PR. But this example is mostly based on a third party library that I do not want to add as a dependency to this lib

**Comment by MarcDahlem** [comment]
Its just in the examples. Dont you consider this as better than having no code at all available how to store a data series in a useful format?
Which json library would you prefer over gson? Moshi? Jackson?

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangel
 
 ### Added
 - **Enhancement** Added possibility to use CostModels when backtesting with the BacktestExecutor
+- **Example** added a json serialization and deserialization example of BarSeries
```
Comment (team172011) [inline CHANGELOG.md]: Please add using google-gson library

**Review by team172011 — APPROVED** [review]

**Comment by team172011** [comment]
Okay, please resolve conflicts and I will merge this example. I think it adds value to the example project in the same way like plotting data using the jFreeCharts library

File: CHANGELOG.md (hunk 1 above)
Comment (MarcDahlem) [reply CHANGELOG.md]: done

**Review by team172011 — APPROVED** [review]

**Pull request merged**